server/accounts_test.go

```diff
@@ -452,6 +452,12 @@ func TestAccountParseConfigImportsExports(t *testing.T) {
 	if ea.respType != Singleton {
 		t.Fatalf("Expected to get a Singleton response type, got %q", ea.respType)
 	}
+	if want := "latency.tracking.add"; ea.latency.subject != want {
+		t.Fatalf("Expected to get latency subject %q, got %q", want, ea.latency.subject)
+	}
+	if want := int8(100); ea.latency.sampling != want {
```

Comment:
I think 100 is default, let's make sure we test that and setting it specifically.